app/editor/src/features/content/list-view/ContentListView.test.tsx

```diff
@@ -59,67 +59,69 @@ jest.mock('store/hooks', () => ({
   useLookup: () => mockUseLookups,
 }));
 
-it('renders correctly...', () => {
+it('renders correctly...', async () => {
   const { container } = render(
     <TestWrapper>
       <ContentListView />
     </TestWrapper>,
   );
-
-  expect(container).toMatchSnapshot();
+  // TODO: This snapshot isn't waiting for the `findContent` to resolve.
+  await waitFor(() => expect(container).toMatchSnapshot());
```

Comment:
This snapshot doesn't really work correctly because it doesn't wait for the promise to resolve in the `findContent` method.